account_financial_report_qweb/report/open_items.py

```diff
@@ -617,17 +607,19 @@ def _inject_line_values(self, only_empty_partner_line=False):
 ORDER BY
     a.code, ml.date, ml.id
             """
+        full_reconcile_date = fields.Datetime.to_string(fields.Datetime.from_string(self.date_at) + timedelta(days=1))
         self.env.cr.execute(
             query_inject_move_line,
             (self.date_at,
-             self.date_at,
              self.id,
-             self.date_at,
+             full_reconcile_date,
              self.date_at,
              self.id,
+             full_reconcile_date,
              self.env.uid,
              self.id,
-             self.date_at,)
+             self.date_at,
+             )
```

Comment:
Juste a side note (no need to change): in such  big queries, using `%(name)s` and passing a dict to `execute` is much easier to comprehend and maintain.